index.css

```diff
@@ -5,25 +5,37 @@ body {
 }
 
 .banner {
-	background-color: #000000;
+	background-color: #000;
 }
 
-.aligned {
-	padding-left: 20%;
+nav div {
+	text-align: center;
+}
+
+.logo {
+	max-width: 80%;
+	height: auto;
```

Comment:
Maybe in the future we can add the other website's logo system that uses the sub banner with a \n